src/clr-core/button/button.element.scss

```diff
@@ -1,35 +1,55 @@
+// Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@import './../styles/tokens/generated/index';
+
 @import './../styles/mixins/utils';
+@import './../styles/mixins/mixins';
```

Comment:
The one mixin file and tokens file are already in the base component styles so they do not need to be imported. We should add the other mixin file to the base style or combine the mixin files. 

https://github.com/vmware/clarity/blob/master/src/clr-core/common/base/base.element.scss#L5